### PR TITLE
Add notebook for TransformQAHuggingFaceJsonFormatConfig

### PR DESCRIPTION
**Pull request by frank-suwen**
Add the notebook for HuggingFace to use format in json #110

### CODE REVIEW

**Comment by goldmermaid** [comment]
Hey @frank-suwen can you address this PR based on our conversation offline? Thanks!

File: example/transform/huggingface_model_json.ipynb [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.10.13"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2
+}
```
Comment (CambioML) [inline example/transform/huggingface_model_json.ipynb:391]: nit: let's add the footer as the other notebook to include CambioML.

File: example/transform/huggingface_model_json.ipynb [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+   "source": [
+    "Congrats! Your question answers from the given knowledge context are generated!\n",
+    "\n",
+    "\n",
+    "---\n",
+    "\n",
+    "\n",
+    "## Appendix\n",
+    "\n",
+    "We benchmarked to see the optimal `batch_size` for the `TransformQAHuggingFaceJsonFormatConfig` flow. The answer is \"It depends on your data token length, your GPU memory, your LLM size, etc.\" In the following experiment, we use a GPU with 24G memory and a quantized LLM (2G). We still use the above 400 raw data strings `raw_context_input_400`.\n",
```
Comment (CambioML) [inline example/transform/huggingface_model_json.ipynb:344]: You might also want to include what compute resource you are using such as AWS instance type.

**Review by CambioML — APPROVED** [review]
approved with minor comment to address.

**Comment by goldmermaid** [comment]
LGTM

**Pull request merged**